src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -39,14 +39,16 @@ class FailedMessagesRetryCommand extends AbstractFailedMessagesCommand
     private $eventDispatcher;
     private $messageBus;
     private $logger;
+    private $globalReceiverName;
```

Comment:
Is this argument ever used?

Reply:
This is to not to break BC, so we know which is the default transport, in case multiple are defined.

Reply:
But, this property is never *used* in this class. If you look at the final source of this class, and search for `$this->globalReceiverName`, we set it in the constructor, but never reference it again. Or am I missing something?